src/comparator/message_comparator.py

```diff
@@ -139,88 +138,3 @@ def _compare_nested_enums(self, nested_enum_dict_original, nested_enum_dict_upda
                 nested_enum_dict_update[name],
                 self.finding_container,
             ).compare()
-
-    def _compare_resources(
-        self,
-        resource_original,
-        resource_update,
-    ):
-        if not resource_original and not resource_update:
-            return
-        # 1. A new resource definition is added.
```

Comment:
When a new resource annotation is added in message-level, that means in the global database, a new resource is added, which should be checked in the file-set comparator.